projects/ghc-performance-dashboards.markdown

```diff
@@ -0,0 +1,14 @@
+---
+status: ideation
+title: GHC Performance Dashboards
+summary: The Haskell Foundation is working with the GHC team to help provide dashboard monitors for Haskell critical libraries (e.g. Aeson, Text, Bytestring).
+link: https://gitlab.haskell.org/ghc/ghc
+link-text: https://gitlab.haskell.org/ghc/ghc
+subtitle: Project Leaders
+leader0name: Ben Gamari
+leader0img: /assets/images/project-leads/bg.png
```

Comment:
Something I was not sure about was putting full image paths vs. knowing where they have to be for the context and just having the name.  So:

`leader0img: /assets/images/projectleads/bg.png`

vs. 

`leader0img: bg.png`

where the usage prepends the canonical path depending on the usage context. I went with the former but I think I'm leaning towards the latter. Not something we need to fix for this PR, just thought about it again while review.

Reply:
problem with this is that some comes from "project-leads" and others come from "board-members" so it won't be only the name

```
leader0img: bg.png
leader1img: thc.png
```
vs
```
leader0img: project-leads/bg.png
leader1img: board-members/thc.png
```
We can image to separate the two properties : 

```
leader0team: project-leads
leader0img: bg.png
leader1team: board-members
leader1img: thc.png
```
and to build it on use in project template ```project.html```
```
<img src="$leader0team$/$leader0img$" alt="$leader0name$">
```
but it's maybe less straightforward than full path or team + name partial path

Reply:
I let you look at the current proposal (compromise :smile:) @myShoggoth